AIProxyStabilityAI/AIProxyStabilityAI/ImageGenView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ImageGenView.swift
+//  AIProxyStabilityAI
+//
+//  Created by Todd Hamilton on 8/13/24.
+//
+
+import SwiftUI
+import AIProxy
+
+struct ImageGenView: View {
+    
+    @State private var prompt = ""
+    @State private var image: UIImage? = nil
+    @State private var isLoading = false
+    
+    func generate() async throws {
+        isLoading = true  // Start loading
+        defer { isLoading = false }
+        do {
+            let body = StabilityAIUltraRequestBody(prompt: prompt)
+            
+            // To use image-to-image the following parameters are required:
```

Comment:
Could maybe prefix this by saying "This demo is of `text-to-image`, which only requires a prompt". To use `image-to-image`, the following...."